docs/index.html

```diff
@@ -437,6 +466,7 @@ <h2>#TwitterThrowback</h2>
                     }
                     return all;
                 }, {});
+                console.log(results);
```

Comment:
Could you remove console.log from the PR? @MichaelGee

Reply:
Yeah. Sorry about that. I missed that part